community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/standard/NodeRecordFormat.java

```diff
@@ -32,6 +30,7 @@
 {
     // in_use(byte)+next_rel_id(int)+next_prop_id(int)+labels(5)+extra(byte)
     public static final int RECORD_SIZE = 15;
+    private static final int HEADER_SIZE = 1;
```

Comment:
it looks like this is not a right place for this constant, all of the one byte header records will have header size of 1, not only node records